src/scripts/pokemons/PokemonHelper.ts

```diff
@@ -1,789 +1,3 @@
-///<reference path="../GameConstants.d.ts"/>
-
-enum PokemonLocationType {
-    Route,
-    Roaming,
-    Dungeon,
-    DungeonBoss,
-    DungeonChest,
-    Evolution,
-    Egg,
-    Baby,
-    Shop,
-    Fossil,
-    Safari,
-    BattleFrontier,
-    Wandering,
-    Discord,
-    QuestLineReward,
-    TempBattleReward,
-    GymReward,
-    DungeonReward,
-    Trade,
-    GiftNPC,
-    ShadowPokemon,
-    DreamOrb,
-    BattleCafe,
-    SafariItem
-}
-
 class PokemonHelper extends TmpPokemonHelper {
```

Comment:
Do we still need this file?

Reply:
Ah if we remove this file then we need to add a bunch of refs:
```ts
/// <reference path="../declarations/pokemons/PokemonHelper.d.ts" />
```
Will merge as is for now as there is a lot of refs for it.